src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -186,11 +186,12 @@ paths:
     $ref: 'paths/source_project_name_cmd_addchannels.yaml'
   /source/{project_name}?cmd=createpatchinfo:
     $ref: 'paths/source_project_name_cmd_createpatchinfo.yaml'
-
```

Comment:
Is this newline intended? :thought_balloon:

Reply:
Yes, it's an intentional change. All these paths are related and there was this empty line available, probably because of conflicts in the previous merge.